pkgs/development/python-modules/torch/default.nix

```diff
@@ -178,6 +181,10 @@ in buildPythonPackage rec {
         'message(FATAL_ERROR "Found NCCL header version and library version' \
         'message(WARNING "Found NCCL header version and library version'
   ''
+  # TODO(@connorbaker): Remove this patch after 2.1.0 lands.
+  + lib.optionalString cudaSupport ''
+    substituteInPlace torch/utils/cpp_extension.py --replace "'8.6', '8.9'" "'8.6', '8.7', '8.9'"
```

Comment:
RE: readability

Maybe we could do with a `.patch` or

```bash
substituteInPlace torch/utils/cpp_extension.py \
  --replace \
    "'8.6', '8.9'" \
    "'8.6', '8.7', '8.9'"
```

Reply:
Fixed in latest -- I went with the latter instead of introducing a new patch file (mostly because I feel it sticks out visually and I don't want to have to generate a patch file).